sematic/ui/packages/common/src/hooks/runHooks.ts

```diff
@@ -75,7 +75,7 @@ export function useFiltersConverter(filters: AllFilters | null) {
         if (filters[FilterType.TAGS]) {
             const statusFilters = convertTagsFilterToRunFilters(filters[FilterType.TAGS] as StatusFilters[]);
             if (statusFilters) {
-                conditions.push(statusFilters);
```

Comment:
Was this a bug?

Reply:
Not really. Previously it returned a single entry only. Now it's a list.